src/agor/tools/dev_tooling.py

```
@@ -690,6 +955,161 @@ def prepare_prompt_content(content: str) -> str:
+    except Exception as e:
+        memory_branch = None
+        print(f"⚠️ Memory sync not available: {e}, proceeding without memory branch")
+
+        # Attempt to restore original branch if we have it
+        if original_branch:
+            try:
+                import subprocess
+                subprocess.run(['git', 'checkout', original_branch],
+                             capture_output=True, text=True, check=True)
+                print(f"🔄 Restored original branch: {original_branch} after error")
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)